CodingTracker.AhmadJer99/CodingTracker/Controllers/GenerateRandomData.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Configuration;
+using CodingTracker.Models;
+namespace CodingTracker.Controllers;
+
+internal static class GenerateRandomData
+{
+    public static int RandomRowsNumber { get; set; }
+    private readonly static string? dateFormat = ConfigurationManager.AppSettings.Get("CorrectDateFormat");
+    private readonly static string? timeFormat = ConfigurationManager.AppSettings.Get("CorrectTimeFormat");
+    private readonly static Random gen = new();
```

Comment:
🟠 **Thread-Safe Random**

💡 From .NET 6 you can use `Random.Shared` to get a thread-safe instance of `Random`.

🔧 To get a random integer you just need to do: `int number = Random.Shared.Next();`